tedana/io.py

```diff
@@ -1,64 +1,245 @@
 """
-Functions to handle file input/output
+=============================
```

Comment:
Whoops, didn't realize that would render that way. Do you think it would be good to just ditch what's basically a table of contents up top, and keep it to comments above each section?

Reply:
I'd perhaps use `See Also` sections in the docstrings and/or directly incorporate this info into `api.rst`. I am not sure how Pythonic module-level comments are.